Backend/Models/Users/UserManager.cs

```diff
@@ -17,13 +17,23 @@ public static void Mute(User user, string reason = "")
 
         public static void Unmute(User user) { }
        
-        public static void Kick(User user) 
+        public static void Kick(User user, string reason = "") 
         {
-            UserManager.Disconnect(user);
+            Disconnect(user);
         }
```

Comment:
`reason` is not used here? i guess there should be an alert?

Reply:
Good Catch